cmd/helm/status.go

```diff
@@ -53,6 +53,9 @@ func newStatusCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 				return err
 			}
 
+			// strip chart metadata from the output
```

Comment:
Can you add a FIXME note.  We should come up with a better solution for dumping the chart with the release.

Reply:
Just noticed the issue you created :sheep: